examples/flutter_gallery/test_driver/scroll_perf_web_test.dart

```diff
@@ -12,7 +12,7 @@ void main() {
     FlutterDriver driver;
 
     setUpAll(() async {
-      driver = await FlutterDriver.connect(browser: true);
+      driver = await FlutterDriver.connect();
```

Comment:
You could expose a "supportsTImeline" getter on the driver, then you could wrap the TimelineSummary in that and merge this back with the other test script

Reply:
Discussed with Jonah offline. Going to keep this as it is for now so that we can verify flutter driver works with non-chrome browsers (testing on non-chrome browsers will throw exceptions when calling timeline-related actions).